ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -6527,6 +6527,261 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cluster: api.ci
+  decorate: true
+  interval: 12h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-env: remote-libvirt
+    job-release: "4.3"
+    job-test: e2e
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-origin-installer-e2e-remote-libvirt-ppc64le-4.3
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --kubeconfig=/etc/apici/kubeconfig
+      - --secret-dir=/usr/local/e2e-remote-libvirt-cluster-profile
+      - --secret-dir=/usr/local/pull-secret
+      - --target=e2e-remote-libvirt
+      - --template=/usr/local/e2e-remote-libvirt
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.3-ci
+      - name: BRANCH
+        value: "4.3"
+      - name: ARCH
+        value: ppc64le
+      - name: CLUSTER_TYPE
+        value: libvirt-ppc64le
+      - name: BASTION_SVC
+        value: sshd
+      - name: BASTION_SVC_IP
+        value: 172.30.168.41
+      - name: REMOTE_LIBVIRT_HOSTNAME
+        value: c155f2u33
```

Comment:
The purpose of this variable is to ensure that the dns name for the libvirt service is the same from the test container's perspective as well as the installation hyperviser's perspective. This is so that the machine-config-operator can bring up additional workers, but it doesn't seem like it's been a problem for the power deployments. It was an issue we ran into during Z dev though.

If this value is in fact arbitrary, I think we should give it a simpler name.
`ocponp` or `ocponpower` are both said names that would be parallel to our current Z configuration

`c155f2u33` seems like it's a random hash. Maybe we want to set this to something more

Reply:
I'm guessing the lab guys named it based on its location in the lab. I don't think it actually resolves, so I'll try the arbitrary `ocponpower`.

Reply:
It's OK if it's actually a representation of a lab location. It just wasn't clear at first glance. :) Up to you.